cross/cffi/Makefile

```diff
@@ -13,14 +13,4 @@ LICENSE  = MIT
 
 PLIST_TRANSFORM = sed -e 's%@PYTHON_SITE_PACKAGES@%$(PYTHON_LIB_DIR)/site-packages%' -e 's%@PYTHON_VERSION@%$(SPK_SHORT_VERS)%'
 
-POST_INSTALL_TARGET = install_into_cross_env
```

Comment:
It turns out that, once installed in the native python3 environment this was no longer required.